src/common/cockpitsystem.c

```diff
@@ -90,6 +94,48 @@ cockpit_system_load_os_release (void)
   return result;
 }
 
+JsonObject *
+cockpit_system_load_os_release_to_json (void)
```

Comment:
Can we make these two functions symmetrical? That is:

```
JsonObject *
cockpit_system_encode_os_release (GHashtable* os_release,
                                                            const gchar **field_names)

GHashtable *
cockpit_system_decode_os_release (JsonObject *object);
```

As you can see, the caller should also specify the list of fields. Then these functions can be used by ```src/ws/cockpithandlers.c```